### ISSUE TITLE
With Postman

### ISSUE THREAD

**Issue by vikram002** [microsoft/Graphir#96]
1) What value should the client Id and Scope should be while calling the graphir from Postman.
2) Even after receiving the token with the user details from Azure AD; there is still Msal exception being thrown while triggering the request from the  Banana POP Ui with valid token values.

**Comment by nirzaf** [comment]
Client Id should be the App Registration's client Id and the scope should be {Your FHIR Base URL}/user_impersonation, you can define your own scope instead of user_impersonation. 

Make sure you have defined the audience properly

**Comment by vikram002** [comment]
App Registration complete. 
Client App Registration complete.

Postman setup done accordingly; which is mentioned here. Added client id(App registration) . 
I get the token now ; however when I pass the same from the Banana Pop UI settings and hit run ; it throws MSAL exception.

IDW10502: An MsalUiRequiredException was thrown due to a challenge for the user. See https://aka.ms/ms-id-web/ca_incremental-consent. ",

**Comment by vikram002** [comment]
Yes , I can do that via postman but not using the banana pop ui.

**Comment by vikram002** [comment]
The token which is generated does not have the scope of the GraphAPI - user_impersonation. It only pulls the Microsoft.Graph scopes even after following all the steps mentioned in this repo.

**Comment by pjirsa** [comment]
@vikram002 double check appsettings.json and make sure the `FhirConnection` section has the correct scope

```
"FhirConnection": {
    "BaseUrl": "https://chestist-fhir-api.azurehealthcareapis.com",
    "Scopes": "https://chestist-fhir-api.azurehealthcareapis.com/user_impersonation",
    "UseAuthentication": true,
    "ResultsLimit": "100"
  },
```
The scope **MUST** have the same root URL as your FHIR API.
Then, make sure that the app registration used in the AzureAD section includes permissions to "Azure Healthcare Apis" in the API permissions section.
![image](https://user-images.githubusercontent.com/2207109/206868015-a1e982c4-e915-457f-8877-e71e4ee203b8.png)
